Runtime/WebRTC/NativeWebRtcClient.cs

```diff
@@ -0,0 +1,209 @@
+            }
+
+            if (Logger.IsDebug())
+            {
+                Logger.LogDebug($"New DataChannel: id={id} label={dc.Label}");
+            }
+
+            dcDict.Add(id, dc);
+            idMapper.Add(id);
+            var clientId = idMapper.Get(id);
+
+            // Host only
+            if (peerClient.Role == PeerRole.Host)
+            {
+                dc.OnOpen = () =>
+                {
+                    if (Logger.IsDebug())
+                    {
+                        Logger.LogDebug($"{nameof(dc.OnOpen)}: clientId={clientId}");
+                    }
+                    FireOnConnected(clientId);
+                };
+            }
+
+            // Both Host and Client
+            dc.OnMessage = message => FireOnDataReceived(clientId, Encoding.ASCII.GetString(message));
+            dc.OnClose = () =>
+            {
+                if (Logger.IsDebug())
+                {
+                    Logger.LogDebug($"{nameof(dc.OnClose)}: clientId={clientId}");
+                }
+
+                if (peerClient.Role == PeerRole.Host && disconnectedRemoteClients.Remove(clientId))
+                {
+                    return;
+                }
+                FireOnDisconnected(clientId);
+            };
+        }
+
+        private void ClosePc(string id)
+        {
+            if (!dcDict.TryGetValue(id, out var pc))
+            {
+                return;
+            }
+            pc.Close();
+            dcDict.Remove(id);
+            idMapper.Remove(id);
+        }
+
+        /// <inheritdoc/>
+        protected override async UniTask DoConnectAsync()
+        {
+            if (Logger.IsDebug())
+            {
+                Logger.LogDebug($"{nameof(Connect)}: role={peerClient.Role}");
+            }
+
+            if (peerClient.Role == PeerRole.Client)
+            {
+                var hostId = peerClient.HostId;
+                await UniTask.WaitUntil(() => idMapper.Has(hostId), cancellationToken: cancellation.Token);
+                var clientId = GetHostId(nameof(Connect), hostId);
+                if (!IsHostIdNotFound(clientId))
+                {
+                    FireOnConnected(clientId);
+                }
+            }
+        }
+
+        private const ulong HostIdNotFound = 0;
+        private static bool IsHostIdNotFound(ulong hostId) => hostId == HostIdNotFound;
+
+        [SuppressMessage("Usage", "CC0014")]
+        private ulong GetHostId(string caller, string hostId)
+        {
+            ulong result;
+            if (hostId is not null)
+            {
+                result = idMapper.Has(hostId) ? idMapper.Get(hostId) : HostIdNotFound;
+            }
+            else
+            {
+                result = HostIdNotFound;
+            }
+            if (Logger.IsDebug() && caller != nameof(Send))
+            {
+                Logger.LogDebug($"{nameof(GetHostId)}: caller={caller} hostId={hostId}");
+            }
+            return result;
+        }
```

Comment:
```suggestion
        private ulong GetHostId(string caller, string hostId)
        {
            if (Logger.IsDebug() && caller != nameof(Send))
            {
                Logger.LogDebug($"{nameof(GetHostId)}: caller={caller} hostId={hostId}");
            }
            return hostId is not null && idMapper.Has(hostId) ? idMapper.Get(hostId) : HostIdNotFound;
        }
```

こちらの方がすっきりして良いと思います。

Reply:
fixedhttps://github.com/extreal-dev/Extreal.Integration.Multiplay.NGO/pull/21/commits/e8f4f198f53006f522703946fc5979619e9b9f18

Reply:
確認しました！